config/spandx.config.js

```diff
@@ -1,12 +1,12 @@
-/*global module, process*/
+/*global process*/
```

Comment:
As we update this file, I think we should follow the same pattern that the template frontend starter api does and move this file to a new directory, profiles/local-frontend, like here: https://github.com/RedHatInsights/insights-frontend-starter-app/blob/master/profiles/local-frontend.js - just to keep this consistent across apps.

Reply:
I will take a look after the tests are working properly. :+1: